src/generators/bashHelper.js

```diff
@@ -1,36 +1,37 @@
-import {
-  getFullNetworkPath,
-} from './networkCreator'
-import {
-  copyFile,
-  libRootDir,
-  writeFile,
-} from '../utils/fileUtils'
+import { getFullNetworkPath } from './networkCreator'
```

Comment:
do we care about dependency cycles? - saying we have one here and with ./transactionmanager

Reply:
Yeah there are a few, but they don't seem to cause any issues and i think we can refactor a bit to fix them. Specifically moving the bash script generation out of the bashHelper file. But that was too much for this PR